Documentation/cmdref/cilium-agent.md

```diff
@@ -42,7 +42,7 @@ cilium-agent [flags]
       --bpf-lb-maglev-hash-seed string                       Maglev cluster-wide hash seed (base64 encoded) (default "JLfvgnHc2kaSUFaI")
       --bpf-lb-maglev-table-size uint                        Maglev per service backend table size (parameter M) (default 16381)
       --bpf-lb-map-max int                                   Maximum number of entries in Cilium BPF lbmap (default 65536)
-      --bpf-lb-mode string                                   BPF load balancing mode ("snat", "dsr", "hybrid") (default "snat")
+      --bpf-lb-mode string                                   BPF load balancing mode ("snat", "dsr", "hybrid", "dsr-tunl") (default "snat")
```

Comment:
From a user perspective, what does this option enable?

My understanding is that it enables DSR (specifically, DSR using IPIP) to work when vxlan tunneling is enabled (`--tunnel vxlan`). Assuming, my understanding is correct, do we need such a user-face agent option? Couldn't we do what needs to be done when both `--bpf-lb-mode dsr`and `--tunnel vxlan` is set?

Reply:
I understand that DSR mode and tunl mode cannot be used together.

* https://docs.cilium.io/en/v1.9/gettingstarted/kubeproxy-free/#direct-server-return-dsr
```
DSR currently requires Cilium to be deployed in Native-Routing, i.e. it will not work in either tunneling mode.
```
* https://github.com/cilium/cilium/blob/master/daemon/cmd/kube_proxy_replacement.go#L331

Reply:
@changwoo-nam thanks for working on this PR! I think for the `bpf-lb-mode` we could just reuse `dsr`, and then `bpf-lb-dsr-dispatch` would be set to `ipip`. Right now this option is disallowed in K8s CNI mode, so nothing would break from a reuse perspective.

Reply:
Did you also check whether this can terminate Cilium standalone in L4LB mode?